test/e2e/run_e2e.sh

```diff
@@ -114,6 +119,11 @@ if [ "$create_cluster" = true ]; then
 
         # Wait for the resources to be ready
         ./wait-for-update.py --timeout 1200
+
+        # provision and start load test
+        echo "provision and start load test"
+        ZONE=$(echo "${API_SERVER_URL}" | awk -F . '{print substr($0, index($0,$2))}')
```

Comment:
```sh
export API_SERVER_URL="https://${LOCAL_ID}.${HOSTED_ZONE}"
```
`ZONE` and `HOSTED_ZONE` are the same, aren't they?

Reply:
fixed